addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -681,6 +734,60 @@ var ImageWidget = FileWidget.extend({
         this.el.removeEventListener('load', this._onAttachmentImageLoad, true);
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    async fetchAttachments(number, offset) {
+        if (this.needle && this.searchService !== 'database') {
+            number = this.MAX_DB_ATTACHMENTS;
+            offset = 0;
+        }
+        const result = await this._super(number, offset);
+        // Color-substitution for dynamic SVG attachment
+        const primaryColor = getCSSVariableValue('o-color-1');
+        this.attachments.forEach(attachment => {
+            if (attachment.image_src.startsWith('/')) {
```

Comment:
I reported a bug on discord a few day ago when you click on some static image where image_src was null, it has been processed and fixed ?

Reply:
Yes, that was caused by attachments being created with the wrong mimetype when not logged in as admin, they were not considered images and hence had no image_src, but this has been fixed.